browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -902,6 +902,15 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_P3A_INSIGHTS" desc="Text for p3a insights onboarding">
         Share private and anonymous product insights about what features are being used by Brave's users.
       </message>
+      <message name="IDS_WDP_TITLE" desc="Text for WDP title onboarding">
+        Help Brave Search grow
+      </message>
+      <message name="IDS_WDP_TEXT" desc="Text for WDP text onboarding">
+        Opt into the Web Discovery Project to help grow the index and improve search results. Learn more.
```

Comment:
Is there an actual link that needs to be set if person taps `Learn more`? Where would that go?

Reply:
I have talked to Tim about it and I am going to update it with hyperlink.

Reply:
I noticed when working on the design with Fiona spec.